framework-platform/framework-platform.gradle

```diff
@@ -20,8 +20,8 @@ dependencies {
 	api(platform("org.apache.groovy:groovy-bom:4.0.17"))
 	api(platform("org.apache.logging.log4j:log4j-bom:2.22.1"))
 	api(platform("org.assertj:assertj-bom:3.25.2"))
-	api(platform("org.eclipse.jetty:jetty-bom:12.0.7-SNAPSHOT"))
-	api(platform("org.eclipse.jetty.ee10:jetty-ee10-bom:12.0.7-SNAPSHOT"))
```

Comment:
oops I should not have pushed that change.  We probably should also remove the localRepository changes.

Reply:
Yup. we probably need to have 12.0.7 released first.
You should probably turn https://github.com/spring-projects/spring-framework/pull/32097 to a draft PR as this will not be ready before 12.0.7 and then we can cleanup some changes